wis2box-management/wis2box/data/__init__.py

```diff
@@ -201,11 +203,10 @@ def add_collection(ctx, filepath, verbosity):
     """Add collection index to API backend"""
 
     meta = gcm(filepath.read())
-    # topic_hierarchy = meta['id']
 
-    # th, _ = validate_and_load(topic_hierarchy)
+    if meta['topic_hierarchy'] not in DATADIR_DATA_MAPPINGS['data'].keys():
+        raise click.ClickException('No matching topic hierarchy')
```

Comment:
exception = f"topic_hierarchy={meta['topic_hierarchy']} not found in data-mappings\n"
exception += f"topic_hierarchy defined in data-mappings: {DATADIR_DATA_MAPPINGS['data'].keys()}" 
raise click.ClickException(exception)